source/__tests__/integration/export-profile.test.js

```diff
@@ -34,7 +35,7 @@ describe('Profiles Export', () => {
 
   describe('exports cleanly', () => {
     beforeAll(async () => {
-      await page.goto('http://127.0.0.1:8000/#/profile/create/')
+      return await page.goto('http://127.0.0.1:8000/#/profile/create/')
```

Comment:
FYI, @mjgiarlo and @jmartin-sul, when I did this without the await (just `return page.goto ...`), I got messages in the console about unresolved promises, or some such.  This looks ... weird ... but it seems to work most of the time.  I decided to keep using this syntax.

Perhaps someone will figure out a more deterministic pattern down the road.

Reply:
weird.  but sure.  i still don't have a solid enough grasp on promises/async/await to have ideas.